### DISCUSSION TITLE
[Hackathon] Ideation for following event(s)

### DISCUSSION THREAD

**Discussion by mspiekermann** [eclipse-edc/.github#13] [Planning]
After the great event and success of the first EDC-Hackathon end of August, we are discussing possible follow-up events, e.g. again in the context of the Gaia-X Hackathon series.

Here we would like to collect and discuss your ideas and topics that might be addressed in future events.

**Comment by DominikPinsel** [comment]
Maybe the next step, after adding identities in the last Hackathon, would be to create Verifiable Credentials (instead of Contracts) and initiate Data Exchange based on them.

**Comment by jimmarino** [comment]
A few things we can demo:

1. Web DID support
2. Federated Catalog
3. IDS and DAPS support

Our new UI person may also be able to create a simple UI for the Hackathon as well.
